### PR TITLE
Fix feedback modal

### PR DESCRIPTION
**Pull request by carylwyatt**
Added a date-time variable to each form as a unique ID to get around having the same form on the page multiple times. 

~I don't think all of my modal issues are fixed, but I'd like to test this much on preview.~ Scratch that, I fixed it! 

The first two commits in here are related to working with my local wordpress environment. I think I can delete them if necessary? I didn't actually change anything in `vite.config.js` but my linter changed all the punctuation 🙃  In `utils.js`, I just added an additional if/else to catch my localhost domain (but again, it looks like a lot because of the linting).

### CODE REVIEW

File: src/js/components/FeedbackFormBasic/index.svelte [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <script>
   import { slide } from 'svelte/transition';
+  export let id = `id${new Date().getTime()}`;
```
Comment (carylwyatt) [inline src/js/components/FeedbackFormBasic/index.svelte:3]: Thanks for this! 💯

File: src/js/components/Navbar/index.svelte [hunk 2]
```diff
@@ -44,11 +44,15 @@
     }
   }
 
-  function openFeedback() {
-    if (location === 'catalog') {
+  function openFeedback(formLocation) {
```
Comment (carylwyatt) [inline src/js/components/Navbar/index.svelte:47]: I don't know, this probably could use a refactor... I originally wanted to pass the location in as the form type but I kept getting weird errors, so it's kind of a mess as-is. 🤷‍♀️

File: src/js/lib/utils.js [hunk 3]
```diff
@@ -41,7 +42,10 @@ function setDomains() {
       HT.catalog_domain = hostname;
       HT.www_domain = hostname;
       HT.cookies_domain = 'localhost';
-    } else if ( hostname.indexOf('phiredevelopment') > -1 ) {
+    } else if (hostname.indexOf('phiredevelopment') > -1) {
+      // shameless green
+      HT.www_domain = hostname;
+    } else if (hostname.indexOf('phire') > -1) {
```
Comment (carylwyatt) [inline src/js/lib/utils.js:48]: What I actually changed in this file.

**Review by respinos — APPROVED** [review]
Ship it!

**Pull request merged**